frontend/src/app/main/data/team.cljs

```diff
@@ -65,6 +65,10 @@
 (defn fetch-members
   []
   (ptk/reify ::fetch-members
+    ptk/UpdateEvent
+    (update [_ state]
+      (dissoc state :profiles))
+
```

Comment:
I'm not clearly convinced by this change. I mean, the `members-fetched` explicitly merges the fetched profiles with the existing ones, if you dissoc and then fetch, you leave penpot with no profiles on state meanwhile the request is in progress, if request for some reason lags a bit, this can cause a strange behavior.

I think we have two solutions here:
- Don't merge it on `members-fetched` and directly reassign it.
- Use the team members from the team object when we really need that filter

![image](https://github.com/user-attachments/assets/994c0100-4516-42ad-9214-59f488d4d669)